community/primitive-collections/src/main/java/org/neo4j/collection/primitive/PrimitiveIntCollections.java

```diff
@@ -826,4 +827,32 @@ public static boolean contains( int[] values, int candidate )
                     " when adding items uniquely to a collection:" + collection );
         }
     }
+
+    /**
+     * Deduplicates values in the sorted {@code values} array.
+     *
+     * @param values sorted array of int values.
+     * @return the provided array if no duplicates were found, otherwise a new shorter array w/o duplicates.
+     */
+    public static int[] dedup( int[] values )
```

Comment:
Oh, then that was duplicated from NodeItemHelper, which this `dedup` method was moved from. Will deduplicate that method as well :)